copulas/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "copulas" %}
+{% set version = "0.5.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/copulas-{{ version }}.tar.gz
+  sha256: 23158761c69e506a1cef3fe601fcca07c05f5e555744bd24bb3c70309e04881d
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - copulas=copulas.cli:main
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - pytest-runner >=2.11.1
+    - python >=3.6,<3.9
+  run:
+    - matplotlib-base >=3.2.0,<4
+    - numpy >=1.18.0,<2
+    - pandas >=1.0,<1.1.5
+    - python >=3.6,<3.9
+    - scipy >=1.4.1,<2
+
+test:
+  imports:
+    - copulas
+    - copulas.bivariate
+  commands:
+    - pip check
+    - copulas --help
+  requires:
+    - pip
+
+about:
+  home: https://github.com/sdv-dev/Copulas
+  summary: A python library for building different types of copulas and using them for sampling.
+  license: MIT
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - conda-forge/rdt
```

Comment:
I believe this is the way to add the maintainers from another related library. If not I can list them individually